### ISSUE TITLE
Ion tokens given from ion projector are immediately removed

### ISSUE THREAD

**Issue by boredompwndu** [baranidlo/bench#101]
The ion token is animated to appear and disappear at the same time. The ship and icon do not contain an ion token. The ship does not perform a white 1 straight on the following turn.

![image](https://user-images.githubusercontent.com/8730661/37477788-f0742736-284e-11e8-8a80-5ff163769275.png)

**Comment by Sleet01** [comment]
This issue was caused by the timing being used on ion projector.  The way the engine currently works is, a move is projected, then any collided ships are notified, then the colliding ship executes its maneuver, then the colliding ship "ends" its maneuver, including cleaning up Ion tokens.  So a token was being assigned, then immediately removed because the engine assumed it was assigned in the previous round.

I've updated Ion Projector to listen for endmaneuver events (triggered in the endmaneuver() function *after* Ion Token removal) and assign an ion token to any ship that ends its maneuver touching the Ion Projector ship.  This should solve the issue.
